apis/k8ssandra/v1alpha1/k8ssandracluster_types.go

```diff
@@ -94,6 +94,24 @@ func (in *K8ssandraCluster) HasStargates() bool {
 	return false
 }
 
+// HasReapers returns true if at least one Reaper resource will be created as part of the creation
+// of this K8ssandraCluster object.
+func (in *K8ssandraCluster) HasReapers() bool {
```

Comment:
Looks like this returns `true` even if no Reaper instances are created. In other words, it returns `true` if there is no Reaper instance declared at the dc-level as long as it is declared at the cluster-level. Is that the expectation?

Reply:
Yes. A non-nil cluster-level Reaper means that all dcs will get a Reaper deployment inherited from the cluster-level specs. This is in line with how the corresponding `Coalesce` function is implemented.

Reply:
I added a small unit test to exercise the method's contract.

Reply:
Thanks for the clarification.